<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDlg</class>
 <widget class="QDialog" name="SetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选项</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>329</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>329</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;



;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="theme">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>主题设置</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>115</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>115</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>自定义</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>背景图片</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>360</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>浏览文件夹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>宝石图片</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>360</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>浏览文件夹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>选择编号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>120</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>220</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>七种宝石序号为1-7</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="music">
    <attribute name="title">
     <string>音乐设置</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>9</y>
       <width>115</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>音乐</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>9</y>
       <width>115</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>音效</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>音乐</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>129</y>
       <width>70</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>音效</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>360</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>360</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>浏览文件夹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>130</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(255, 255, 255);
border-style:outset;
border-width:6px;
border-radius:15px;
border-color:rgba(255,255,255,200);
padding:6px;</string>
     </property>
     <property name="text">
      <string>浏览文件夹</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
